extensions/wikia/MyHome/tests/MyHomeTest.php

```diff
@@ -56,6 +56,7 @@ function doEdit($edit) {
 	 * @group UsingDB
 	 */
 	function testNewPageCreation() {
+		$this->markTestSkipped( 'This test is not an unit test' );
 		// set content of new article
 		global $wgParser;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Do not use the global wgParser Parser instance. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWgParserUsage)

[project-id]: wikia:app